pipelines/scipipe/ap_verify.groovy

```diff
@@ -295,31 +295,19 @@ def void verifyDataset(Map p) {
             // Path is partially hard-coded in ap_verify.
             def gen3Dir = util.joinPath(ds.name, 'repo')
             def collection = 'ap_verify-output'
-            util.runGen3ToJob(
-              runDir: runDir,
-              gen3Dir: gen3Dir,
-              collectionName: collection,
-              namespace: '',
-              datasetName: ds.name,
-            )
-            def files = []
-            dir(runDir) {
-              files = findFiles(glob: '**/*.verify.json')
-            }
 
             def codeRef = buildCode ? code.git_ref : "main"
             withEnv([
               "refs=${codeRef}",
             ]) {
-              files.each { f ->
-                util.runDispatchVerify(
-                  runDir: runDir,
-                  lsstswDir: fakeLsstswDir,
-                  datasetName: ds.name,
-                  resultFile: f,
-                  squashUrl: sqre.squash.url,
-                )
-              }
+              util.runVerifyToSasquatch(
+                runDir: runDir,
+                gen3Dir: gen3Dir,
+                collectionName: collection,
+                namespace: "lsst.dm.ap",
```

Comment:
I'm confused by the comment on Jira. Should this be changed to `lsst.verify`, or to `lsst.verify.ap`?

Reply:
Use `lsst.verify.ap` here, this is going to be the prefix used to name your metrics. I'll update the documentation to make it clear.